netlify/functions/newsletter.ts

```
@@ -0,0 +1,950 @@
+
+/**
+ * Sanitizes input string by removing potentially harmful content
+ */
+function sanitizeInput(input: string): string {
+  if (!input || typeof input !== 'string') {
+    return '';
+  }
+
+  return input
+    .trim()
+    .replace(/[\x00-\x1F\x7F]/g, '') // Remove control characters
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)